src/content/en/tools/workbox/guides/using-bundlers.md

```diff
@@ -1,302 +1,94 @@
-using](/web/tools/workbox/guides/get-started#importing_workbox)
-(via `workbox-sw`) when they're referenced. For example, if your code never
-references `workbox.backgroundSync`, then the `workbox-background-sync` package
-will not be downloaded.
+Using Workbox in this fashion allows you to pull in precisely the bits of Workbox that you plan on using, and leads to a smaller overall service worker file. Creating your own bundles of the Workbox runtime libraries also means that you don't need to load the Workbox runtime from third-party web servers.
 
-While only loading the _packages_ being used is a good first step in reducing
-your total service worker file size, you can often reduce the total size even
-more by only loading the specific modules your code imports.
+## Creating a service worker bundle
 
-You can do this using JavaScript bundling tools like
-[webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org) to build
-your own custom service worker file. This guide will outline the recommended way
-to use bundlers with Workbox.
+### Choose a bundler
 
-## Importing workbox code into your bundle
+Any [bundler that supports JavaScript modules](https://bundlers.tooling.report/) will work when bundling the Workbox runtime libraries. Some popular choices include:
 
-When using Workbox without a bundler, you import Workbox into your service
-worker file using
-[`importScripts()`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts):
+- [webpack](https://webpack.js.org/)
+- [Rollup](https://rollupjs.org/guide/en/)
+- [Parcel](https://parceljs.org/)
 
-<pre class="prettyprint js">
-// Import workbox-sw, which defines the global `workbox` object.
-importScripts('{% include "web/tools/workbox/_shared/workbox-sw-cdn-url.html" %}');
+If you're already using a bundler for other parts of your web app, the using the same one for your service worker as part of your existing build process should work fine.
 
-// Then reference the workbox packages you need, which will dynamically
-// call `importScripts()` to load the individual packages.
-workbox.precaching.precacheAndRoute([...]);
-</pre>
+Please consult the documentation for each bundler to learn more about general configuration. There are a few Workbox-specific build options that are explained in the following sections.
 
-When using a JavaScript bundler, you don't need (and actually shouldn't use) the
-`workbox` global or the `workbox-sw` module, as you can import the individual
-package files directly.
+### Write your service worker source code
 
-Here's how you'd import and run the above code using a bundler:
+The following example is a hypothetical service worker source file that imports a few different Workbox libraries. This source file needs to be processed by a bundler before it could be run in a browser.
 
 ```javascript
-// Import just the `workbox-precaching` package.
+// These JavaScript module imports that need to be bundled:
```

Comment:
I think you're missing the word "are" here?

Reply:
I've removed "that" instead.